etc/profile-a-l/ledger-live-desktop.profile

```diff
@@ -0,0 +1,45 @@
+# Firejail profile for Ledger Live
+# This file is overwritten after every install/update
+# Persistent local customizations
+include ledger-live-desktop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/Ledger Live
+
```

Comment:
You forgot all the `disable-*.inc` includes... Follow the template and double-check if the app still has its regular functionalities please.

```
[...]
# remove disable-write-mnt.inc if you set disable-mnt
include disable-common.inc
include disable-devel.inc
include disable-exec.inc
include disable-interpreters.inc
include disable-proc.inc
include disable-programs.inc
include disable-shell.inc
include disable-write-mnt.inc
include disable-X11.inc
include disable-xdg.inc
[...]
```

Reply:
Sorry for this oversight, initially I was including `electron.profile`, but eventually removed it and forgot to add the disables.

I have added them now, will comment on the issues I had in separate threads.

Reply:
`disable-X11.inc` causes the app not to start with this error log:

```
[12:0222/101944.253536:ERROR:ozone_platform_x11.cc(240)] Missing X server or $DISPLAY
```